.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         # not all apps support node versions below, but at the very
         # least they all build
-        apps: [express,fastify3,hapi18,hapi19,hapi20,koa,kraken,loopback,loopback4,restify,sails]
+        apps: [express,fastify3,hapi18,hapi19,hapi20,hapi21,koa,kraken,loopback,loopback4,restify,sails]
```

Comment:
This workflow is for building the images that we use for v4 Screener tests. We need a change in `build_v5_screener_imgs.yml` only, not here